src/datapattern.jl

```diff
@@ -23,9 +23,12 @@ LearnBase.gettarget(::typeof(identity), dt::DataFrameRow) =
     _throw_table_error()
 
 # convenience syntax to allow column name
-LearnBase.gettarget(col::Symbol, dt::AbstractDataFrame) = dt[1, col]
+LearnBase.gettarget(col::Symbol, dt::AbstractDataFrame) =
```

Comment:
I understand that for this to work we should make sure that the data frame has only one row. Right?

Also the question is why do we only allow `Symbol` for `col`. Also integer or string index is allowed in DataFrames.jl.